### PR TITLE
Add support for Postgres Databases

### LINKED ISSUE #33: postgres support
it looks like 'mysql' is hardcoded in the dockerfile.
Can we support other db types like postgres in this container?

https://github.com/tiredofit/docker-freescout/blob/f477098cf232b9d572e1f6868fbefd2232218f69/install/etc/cont-init.d/30-freescout#L180

:)

### PR DESCRIPTION
**Pull request by rustymunkey**
This PR adds support for connecting the freescout application with a postgres database.

This fixes #33 and #95 .

In addition to the changes proposed here, it is still required to enable the php extension `pdo_pgsql`. As far as i understand the setup of the Docker image, this would best be placed in the upstream image [docker-nginx-php-fpm](https://github.com/tiredofit/docker-nginx-php-fpm/blob/master/Dockerfile#L28) by adding:

```
PHP_ENABLE_PDO_PGSQL=TRUE \
```

I tested the modifications successfully by creating a new image based on the `docker-freescout` image with this Dockerfile:

```
FROM tiredofit/freescout:1.15.70

COPY install/etc/cont-init.d/30-freescout etc/cont-init.d/30-freescout
RUN php-ext enable pdo_pgsql
````

However, the setup of freescout does not finish automatically due to a bug or an presumably outdated dependency in the freescout application. I am going to create an issue in the freescout repository and will update this PR afterwards.
By patching the broken functions in-place and manually completing the setup steps, I was able to create a fully functioning freescout instance.

### CODE REVIEW

**Comment by tiredofit** [comment]
Nice one! This is shaping up nicely. Congrats on going through my images to try to figure them out and add to them!

A few notes:

- You can get away with putting your PHP_ENABLE statement right into the Freescout Dockerfile. That will ensure its enabled on each startup for this specific image. When building, you may notice that I execute `php-ext enable core` which loads it during the build process so that things like composer won't fail if it is looking for it.

- When checking for the tables existence (new install) you may run into some issues with passing your Postgres password like that. If it does, you can ommit the password in that string and before you execute the command export the password like this:

`export PGPASSWORD="${DB_TYPE}"`

I've run into this a few times with some of my other images, just some helpful tips to get you more refined.

**Comment by tiredofit** [comment]
OK Well then lets merge this in!

**Comment by tiredofit** [comment]
Tagged as `1.16.0` - Thanks for your contribution!

File: install/assets/defaults/30-freescout [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #!/command/with-contenv bash
 
+DB_TYPE=${DB_PORT:-"mysql"}
```
Comment (Majsvaffla) [inline install/assets/defaults/30-freescout:3]: Isn't this supposed to be `DB_TYPE=${DB_TYPE:-"mysql"}`?

Comment (tiredofit) [reply install/assets/defaults/30-freescout:3]: Ack - yes. Fixing.

Comment (tiredofit) [reply install/assets/defaults/30-freescout:3]: Fixed with `1.16.2`

**Pull request merged**